### PR TITLE
Oadp 2155 rosa sts docs hvider

### PR DESCRIPTION
**Pull request by HagayVider1**
OSDOCS# : OADP2155

Version(s): OADP 1.2.1

Issue: https://issues.redhat.com/browse/OADP-2155

Link to docs preview: https://63294--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws#installing-oadp-aws

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc [hunk 1]
```diff
@@ -18,9 +18,17 @@ You configure AWS for Velero, create a default `Secret`, and then install the Da
 To install the OADP Operator in a restricted network environment, you must first disable the default OperatorHub sources and mirror the Operator catalog. See xref:../../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks] for details.
 
 include::modules/oadp-installing-operator.adoc[leveloffset=+1]
+
 include::modules/migration-configuring-aws-s3.adoc[leveloffset=+1]
+
+include::modules/oadp-preparing-aws-credentials.adoc[leveloffset=+1]
+
+include::modules/oadp-installing-oadp-rosa-sts.adoc[leveloffset=+1]
```
Comment (anarnold97) [inline backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc:26]: @HagayVider1 - I do not think this order of modules makes much sense.

Please ensure you understand the difference between installing the OpenShift API for Data Protection (OADP) with Amazon Web Services (AWS) and Red Hat OpenShift® Service on AWS (ROSA). 

Let us talk about this on Thursday and see if we can improve it as people might fall between the options.

File: rosa_backing_up_and_restoring_applications/backing-up-applications.adoc [hunk 2]
```diff
@@ -39,5 +39,6 @@ include::modules/oadp-installing-oadp-rosa-sts.adoc[leveloffset=+1]
 * link:https://docs.openshift.com/rosa/rosa_getting_started/rosa-sts-getting-started-workflow.html[Getting started with ROSA STS]
 * link:https://docs.openshift.com/rosa/rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.html[Creating a ROSA cluster with STS]
 * link:https://docs.openshift.com/container-platform/4.13/backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.html[About installing OADP]
+* link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/backup_and_restore/application-backup-and-restore#installing-oadp-aws[Installing and configuring the OpenShift API for Data Protection with Amazon Web Services]]
```
Comment (anarnold97) [inline rosa_backing_up_and_restoring_applications/backing-up-applications.adoc:42]: broken link.... look at the end **]]**

File: backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc (hunk 1 above)
Comment (HagayVider1) [reply backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc:26]: Accepted. I'll change this.

Comment (anarnold97) [reply backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc:26]: Let us walk through it later today

**Pull request closed without merging**